test/ergw_test_lib.erl

```diff
@@ -373,3 +374,17 @@ mkint(C) when $A =< C, C =< $F ->
     C - $A + 10;
 mkint(C) when $a =< C, C =< $f ->
     C - $a + 10.
+
+%%%===================================================================
+%%% Exometer helpers
+%%%===================================================================
+
+match_exo_value(Path, Expect) ->
+    Value = get_exo_value(Path),
+    ?equal(Expect, get_exo_value(Path)).
+
+get_exo_value(Path) ->
+    get_value(exometer:get_value(Path)).
```

Comment:
the get_value wrapper is not needed, in the test cases we want to make sure those exometer value exists.

A simple:
```
get_exo_value(Path) ->
    {ok, Value} = exometer:get_value(Path),
    proplists:get_value(value, Value).
```

is enough.